examples/create_postcards_from_csv/postcards.py

```diff
@@ -0,0 +1,152 @@
+
+# Input check
+if len(sys.argv) < 2:
+    print("Please provide an input CSV file as an argument.")
+    print("usage: python letter.py <CSV_FILE>")
+    sys.exit(1)
+
+input_filename = sys.argv[1]
+
+success_csv_fields = [
+    'name',
+    'id',
+    'url',
+    'address_line1',
+    'address_line2',
+    'address_city',
+    'address_state',
+    'address_zip'
+]
+errors_csv_fields = ['error']
+
+# create the output directory,
+output_dir = os.path.join('.',  'output')
+if not os.path.isdir(output_dir):
+    os.mkdir(output_dir)
+
+timestamp = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+
+try:
+    output_dir = os.path.join(output_dir, timestamp)
+    os.mkdir(output_dir)
+except Exception, e:
+    print('Error: ' + str(e))
+    print('Failed to create output directory. Aborting all sends.')
+    sys.exit(1)
+
+# output csv names
+success_filename = os.path.join(output_dir, 'success.csv')
+errors_filename = os.path.join(output_dir, 'errors.csv')
+
+with open('postcard_front.html', 'r') as front_file,\
+        open('postcard_back.html', 'r') as back_file:
+    front_html = front_file.read()
+    back_html = back_file.read()
+
+with open(input_filename, 'r') as input, \
+        open(success_filename, 'w') as success, \
+        open(errors_filename, 'w') as errors:
+
+    # Print mode to screen
+    mode = lob.api_key.split('_')[0]
+    print('Sending postcards in ' + mode.upper() + ' mode.')
+
+    input_csv = csv.DictReader(input)
+    errors_csv_fields += input_csv.fieldnames
+
+    success_csv = csv.DictWriter(success, fieldnames=success_csv_fields)
+    errors_csv = csv.DictWriter(errors, fieldnames=errors_csv_fields)
+
+    err_count = 0
+
+    # Loop through input CSV rows
+    for idx, row in enumerate(input_csv):
+        # Create postcard from row
+        try:
+            postcard = lob.Postcard.create(
+                to_address={
+                    'name': row['name'],
+                    'address_line1': row['address_line1'],
+                    'address_line2': row['address_line2'],
+                    'address_city': row['address_city'],
+                    'address_state': row['address_state'],
+                    'address_zip': row['address_zip'],
+                    'address_country': row['address_country']
+                },
+                from_address=from_address,
+                size='6x11',
+                front=front_html,
+                back=back_html,
+                data={
+                    'background_image': row['background_image'],
+                    'background_color': row['background_color'],
+                    'name': row['name'],
+                    'car': row['car'],
+                    'mileage': row['mileage']
+                }
+            )
+        except Exception, e:
+            error_row = {'error': e}
+            error_row.update(row)
+            errors_csv.writerow(error_row)
+            err_count += 1
+            sys.stdout.write('E')
```

Comment:
I don't understand the point of this 'e'... why not "Error: check error writer for details" or something